### DISCUSSION TITLE
Generate Ascii

### DISCUSSION THREAD

**Discussion by jrock2004** [goolord/alpha-nvim#256] [Q&A]
Not sure if this is considered ascii but where or how do people create these?

![image](https://github.com/goolord/alpha-nvim/assets/655716/20cb91cf-e814-4644-8498-1006952fd7da)

**Comment by goolord** [comment]
i used this https://patorjk.com/software/taag/#p=display&f=Larry%203D&t=neovim and then edited it by hand

**Comment by Asthestarsfalll** [comment]
Hi, I develop a tool to generate such ascii arts with color, see [here](https://github.com/Asthestarsfalll/img2art)
![2024-01-22_22-31](https://github.com/goolord/alpha-nvim/assets/72954905/2403d53e-cf46-4f74-a4b9-009cc39e8fdd)
![2024-01-22_22-31_1](https://github.com/goolord/alpha-nvim/assets/72954905/c02590d3-45db-4b8c-8a1c-92e5704a080a)
![2024-01-22_22-31_2](https://github.com/goolord/alpha-nvim/assets/72954905/99f77f34-da42-45c9-99d6-7060848de453)
![2024-01-22_22-31_3](https://github.com/goolord/alpha-nvim/assets/72954905/9dc1ce0c-be65-4541-ad7e-4c01900d7870)

**Reply by jrock2004** [reply]
That is cool. Guess I need to hunt an anime image now :)
